qemu/tests/cfg/multi_disk.cfg

```diff
@@ -141,6 +140,30 @@
                         stg_params += "drive_port:range(0,255,127) "
                         need_reboot = yes
                         need_shutdown = yes
+        - spapr_vscsi_variants:
+            only spapr_vscsi
+            stg_image_size = 50M
+            stg_params = "drive_format:scsi-hd "
+            variants:
+                - @passthrough:
+                    stg_params = "drive_format:scsi-block "
+                    kill_vm = yes
+                    force_create_image = no
+                    pre_command = "modprobe -r scsi_debug; modprobe sg; modprobe scsi_debug add_host=9 dev_size_mb=50"
+                    post_command = "rmmod scsi_debug"
+                    stg_params += "image_raw_device:yes "
+                    stg_params += "image_format:raw "
+                    stg_params += "indirect_image_select:range(-9,0) "
+                    Linux:
+                        need_reboot = yes
```

Comment:
Suggest that delete the limit of 
```
                    Linux:
```
because spapr_vscsi is not supportted on windows.

Reply:
@YongxueHong Updated and test PASS:

'1-Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.0.1.ppc64le.io-github-autotest-qemu.multi_disk.spapr_vscsi_variants.block' Finshed 2019-05-30 01:47:35 2019-05-30 01:50:17 PASS 162.327087
'2-Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.0.1.ppc64le.io-github-autotest-qemu.multi_disk.spapr_vscsi_variants.generic' Finshed 2019-05-30 01:50:17 2019-05-30 01:53:54 PASS 216.979804

Thanks for the review.